jetty-server/pom.xml

```diff
@@ -4,16 +4,40 @@
     <artifactId>jetty-project</artifactId>
     <version>10.0.0-SNAPSHOT</version>
   </parent>
+
   <modelVersion>4.0.0</modelVersion>
   <artifactId>jetty-server</artifactId>
   <name>Jetty :: Server Core</name>
   <description>The core jetty server artifact.</description>
   <url>http://www.eclipse.org/jetty</url>
+
   <properties>
     <bundle-symbolic-name>${project.groupId}.server</bundle-symbolic-name>
+    <jetty-http-tests-jar>${settings.localRepository}/org/eclipse/jetty/jetty-http/${project.version}/jetty-http-${project.version}-tests.jar</jetty-http-tests-jar>
```

Comment:
This seems awkward.
@olamy got any ideas on how to do this more clearly? and not via the local repository?

Reply:
sadly no...... I would prefer using the directory within reactor (`--patch-module ` can accept directories as well as file but I don't any to say use project.build.outputDirectory of those modules...)

Reply:
I would even prefer using ` <goal>copy-dependencies</goal> ` to copy the jar in the target directory. So at least IDE can deal with that. Otherwise before running test with your IDE you need to first install the jars